examples/shared_samples/hypervisor_manager.rb

```diff
@@ -29,6 +31,8 @@
 # api_version = 1000 & variant = Synergy to OneviewSDK::API1000::Synergy::HypervisorManager
 # api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::HypervisorManager
 # api_version = 1200 & variant = Synergy to OneviewSDK::API1200::Synergy::HypervisorManager
+# api_version = 1600 & variant = C7000 to OneviewSDK::API1200::C7000::HypervisorManager
```

Comment:
It should be :API1600::C7000::HypervisorManager

Reply:
done